nats-server/installation.md

```diff
@@ -74,12 +74,12 @@ Download the zip file matching your systems architecture, and unzip. For this ex
 
 > unzip nats-server.zip -d nats-server
 Archive:  nats-server.zip
-   creating: nats-server-v2.0.0-darwin-amd64/
-  inflating: nats-server-v2.0.0-darwin-amd64/README.md
-  inflating: nats-server-v2.0.0-darwin-amd64/LICENSE
-  inflating: nats-server-v2.0.0darwin-amd64/nats-server
+   creating: nats-server-v2.0.0-linux-amd64/
+  inflating: nats-server-v2.0.0-linux-amd64/README.md
+  inflating: nats-server-v2.0.0-linux-amd64/LICENSE
+  inflating: nats-server-v2.0.0-linux-amd64/nats-server
 
-> cp nats-server-v2.0.0darwin-amd64/nats-server /usr/local/bin
+> sudo cp nats-server-v2.0.0-linux-amd64/nats-server /usr/bin
```

Comment:
I get the `-linux-` change, but I think we should keep `/usr/local/bin`, no?

Reply:
If that is correct then yes - does `local` change if you are using `sudo`?

Reply:
@elucidsoft thoughts on the comments above?

Reply:
local is not as common as /usr/bin for Linux distributions.

Reply:
I believe that user executables should preferably be copied to `/usr/local/bin` and OS executables are usually the ones in `/usr/bin`. But I am fine with the change. Thanks!